
FAIL reset clears to transparent black ctx.reset is not a function. (In 'ctx.reset()', 'ctx.reset' is undefined)

